src/components/client/core/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button } from "components/client";
+import {
+  Accordion as PrimitiveAccordion,
+  AccordionItem,
+  AccordionTrigger,
+  AccordionContent,
+} from "components/primitives";
+import { accordion } from "generated/panda/recipes";
+
+import type { AccordionProps } from "components/primitives";
+import type { ReactNode } from "react";
+
+export interface Props extends AccordionProps {
+  trigger: ReactNode;
+  children: ReactNode;
+  disabled?: boolean;
+  value: string;
+  icon: (props: { isOpen: boolean }) => JSX.Element;
+}
+
+// /**
+//  * Core UI Accordion
+//  */
```

Comment:
To make this valid JSDoc, remove the `//` in front of each line

Reply:
The JSDoc police...

Reply:
:police_officer: :police_officer: